uber/templates/signups/volunteer_agreement.html

```diff
@@ -0,0 +1,94 @@
+(c) neither the Services nor the Work Product will infringe the rights of any third party or violate any law; and<br/>
+(d) Volunteer has full power and authority to enter into this Agreement and to provide the Services in accordance with this Agreement.
+The Company acknowledges and agrees that it has full power and authority to enter into this Agreement.
+</p>
+<p>
+<b>Miscellaneous.</b>
+This Agreement and all performance under this Agreement shall be governed by the laws of the State of California,
+without regard to conflict of law principles. In any dispute relating to this Agreement,
+the parties hereto admit venue and submit themselves to the exclusive jurisdiction of the state or federal courts
+located in Los Angeles County, California. Volunteer's remedy, if any, for any breach of this Agreement shall be
+solely in damages, and Volunteer shall not be entitled to equitable relief by way of injunction or otherwise.
+In the event any provision of this Agreement is determined to be invalid, prohibited or unenforceable by a court or
+other body of competent jurisdiction, this Agreement shall be construed as if such provision has been more narrowly
+drawn so as not to be invalid, prohibited or unenforceable. No waiver of any provision of this Agreement will be
+effective unless it is in writing and signed by both parties, and no such waiver will result in the waiver of any
+other provision of this Agreement. Failure of either party at any time to enforce any of the provisions of this Agreement
+will not be construed as a waiver of such provisions or in any way affect the validity of this Agreement or parts thereof.
+In performing the Services, Volunteer will act and represent itself as an independent volunteer and not as an employee
+or agent of the Company. Volunteer has no authority to obligate the Company by contract or otherwise.
+Volunteer may not assign any rights or obligations under this Agreement without the advance written consent of the Company,
+which may be withheld in its sole discretion. The Company may assign its rights or obligations under this Agreement in its
+sole discretion. This Agreement is the parties' final, exclusive and complete understanding and agreement, and supersedes
+all prior and contemporaneous understandings and agreements relating to the subject matter of this Agreement.
+This Agreement may be amended only by a writing signed by both parties. This Agreement may be executed in counterparts
+which, when taken together, will constitute one Agreement.
+</p>
+
+{% if not attendee.agreed_to_volunteer_agreement %}
+    <form method="post" action="volunteer_agreement">
+        {{ csrf_token() }}
+        <label for="agreed_to_terms">
+          I agree to the terms of the volunteer agreement: <input type="checkbox" id="agreed_to_terms" name="agreed_to_terms" />
+        </label>
+        <br><br>
+        <input class="btn btn-primary" type="submit" value="Save" />
```

Comment:
I made some minor styling tweaks here. Button was hard to see originally. Also made the "I agree to the terms of the volunteer agreement:" clickable.